### PR TITLE
Add performance test for bulk updates

### PR DESCRIPTION
**Pull request by knizhnik**

### CODE REVIEW

**Comment by bayandin** [comment]
If you want to get this test running on CI, you need to add the label `run-benchmarks` to the PR

**Comment by knizhnik** [comment]
> If you want to get this test running on CI, you need to add the label `run-benchmarks` to the PR (and push some new commits or rebase, or close-and-reopen the PR; just rerunning tests won't pick up this label)

Thank you for pointing it.
I know it but ... forget:)

File: test_runner/performance/test_bulk_update.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    env = neon_env_builder.init_start()
+    n_records = 10000000
+
+    timeline_id = env.neon_cli.create_branch("test_bulk_update")
+    tenant_id = env.initial_tenant
+    pg = env.postgres.create_start("test_bulk_update")
+    cur = pg.connect().cursor()
+    cur.execute("set statement_timeout=0")
+
+    cur.execute("create table t(x integer)")
```
[the remark this answers is not in the document]
Comment (knizhnik) [reply test_runner/performance/test_bulk_update.py]: Done

**Comment by knizhnik** [comment]
test_bulk_update.py is merged in main.
Last written LSN changes are merged in vendor/postgres, but postgres version in neon is not yet bumped.

**Comment by vadim2404** [comment]
are you talking about #2912?

**Comment by knizhnik** [comment]
Yes

**Pull request closed without merging**